<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #212121;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="verticalWidget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="List">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
}
QLineEdit:hover
{
	border:1px solid rgb(255,200,0);
}

QLineEdit:selected:active
{
border:1px solid rgb(255,200,0);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FuntionTree" name="treeWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QTreeWidget{  
    background-color: #212121;  
    font-size:20px;  
    color: white;  
    border:2px solid rgb(30,30,30);
}  

QTreeWidget::item{  
    margin:5px 8px 5px 8px;  
	background-color: #212121;  
	background-clip: margin;
	color: white;  
}

QTreeWidget::branch{  
	background-color: #212121;  
 }  
  
QTreeWidget::branch:hover{  
	background: rgb(255, 200, 0);
	color: #212121;
 }  

QTreeView::item:hover {
    background: rgb(255, 200, 0);
color: #212121;
}

QTreeWidget::branch:closed:has-children:!has-siblings,
QTreeWidget::branch:closed:has-children:has-siblings {
    border-image: none;
    image: url(:/icons/icon/branch_close.png)
}
 
QTreeWidget::branch:open:has-children:!has-siblings,
QTreeWidget::branch:open:has-children:has-siblings  {
    border-image: none;
	image: url(:/icons/icon/branch_open.png);
}</string>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>-1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="Cb_Config">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox
{
	border:transparent;   
	color: rgb(255, 255, 255);
    background-color: #212121;  
    padding: 0px 0px 0px 20px;
}
QComboBox:hover
{
	color: #212121;  
    background-color: rgb(255, 200, 0);
}

QComboBox QAbstractItemView{
	background-color: #212121; 
    border:1px solid rgb(109,109,109);
    border:2px;
}
QComboBox::drop-down:disabled {
    	width: 20px;
   	border:transparent; 
	background-color: #212121; 
}
QComboBox::drop-down {
    width: 0px;
    border:transparent; 
    background-color: #212121; 

}
QComboBox::down-arrow {
    padding:10px;
	color: rgb(255, 255, 255);
}
QComboBox QAbstractItemView:item{
    border:transparent; 
	color: rgb(255, 255, 255);
	background-color: #212121; 
	padding: 2px 2px 2px 18px;
	height:22px;
}
QListView::item {
    background: white;
	border:transparent;
}
 
QListView::item:hover {
    	color: #212121;
	background-color:rgb(255, 200, 0);
}


QComboBox QScrollBar::vertical{ /*主体部分*/
    width:5px;
    background-color:rgb(57, 57, 57);
    border:none;
}
QComboBox QScrollBar::handle::vertical{ /*滑块主体*/
    border-radius:5px;
    width: 5px;
    background:rgb(0, 170, 255);
}
QComboBox QScrollBar::handle::vertical::hover{
    background:rgb(50,50,50);
}
QComboBox QScrollBar::add-line::vertical{/*上箭头*/
    border:none;
}
QComboBox QScrollBar::sub-line::vertical{/*下箭头*/
    border:none;
}</string>
            </property>
            <item>
             <property name="text">
              <string>default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bn_Save">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Bn_Save_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
margin:0px 0px 0px 4px
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bn_Save_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
	margin:0px 0px 0px 4px
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bn_Save_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
margin:0px 0px 0px 4px
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bn_Save_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>34</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
margin:0px 0px 0px 4px
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="sW_btn">
         <property name="styleSheet">
          <string notr="true">margin:20px 40px 20px 40px</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget_2" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>-1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bn_image">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
	border:1px solid rgb(50,50,50);
	margin:4px 4px 8px 0px
}
QPushButton:hover{

	border:1px solid rgb(255,200,0);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 200, 0);
	color: rgb(30, 30, 30);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>252</width>
               <height>148</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FuntionTree</class>
   <extends>QTreeWidget</extends>
   <header>funtiontree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
